setup.py

```diff
@@ -44,9 +44,9 @@
         'Development Status :: 5 - Production/Stable',
     ],
     install_requires=[
-        'pyjwt >= 1.0.1',
+        'pyjwt>=2.0',
```

Comment:
I've done some research, and the bump here is actually not needed. The `ExpiredSignatureError` exists in `1.0.1` as you can see [here](https://github.com/jpadilla/pyjwt/blob/1.0.1/jwt/exceptions.py) and [here](https://github.com/jpadilla/pyjwt/blob/1.0.1/jwt/__init__.py).


So, in other words, this change can be reverted, and the only change needed is the change from `ExpiredSignature` to `ExpiredSignatureError` as you've done in `backend.py`.
When this has been done, I'll merge this PR. :+1: